In the first program, the output is 3
In the second program, an error is generated because x is not defined outside of the loop
